apps/prairielearn/src/lib/score-upload.ts

```diff
@@ -246,14 +246,14 @@ export async function uploadAssessmentInstanceScores(
   return serverJob.jobSequenceId;
 }
 
-// missing values and empty strings get mapped to null
+/** missing values and empty strings get mapped to null */
 function getJsonPropertyOrNull(json: Record<string, any>, key: string): any {
   const value = json[key] ?? null;
   if (value === '') return null;
   return value;
 }
 
-// missing values and empty strings get mapped to null
+/** missing values and empty strings get mapped to null */
```

Comment:
This is the same comment as the previous function, maybe it should be removed?

Reply:
Removed in 18a46d5877f5c87a94093a8ab6fa75fb1d7adfba.